composer.json

```diff
@@ -1,4 +1,45 @@
 {
+  "name": "civicrm/civicrm-core",
+  "description": "Open source constituent relationship management for non-profits, NGOs and advocacy organizations.",
+  "type": "library",
+  "authors": [
```

Comment:
It's intended to be humans per the [composer documentation](https://getcomposer.org/doc/04-schema.md#authors), but it's optional and we could even omit it, but pretty much all libraries whose composer.json files I've looked at provide it.

Looking at [Symfony's composer.json](https://github.com/symfony/symfony/blob/master/composer.json), they actually put the project lead and then an entry for the "Symfony Community". That's quite right per the docs, but we could do something similar here? If that's OK with you, I can update my PR!